tests/device_tests/cardano/test_sign_tx.py

```diff
@@ -34,6 +34,7 @@
     "cardano/sign_tx.slip39.json",
 )
 def test_cardano_sign_tx(client, parameters, result):
+    signing_mode = cardano.SIGNING_MODE_IDS[parameters["signing_mode"]]
```

Comment:
```
signing_mode = messages.CardanoSigningMode.__members__[parameters["signing_mode"]]
```

Reply:
https://github.com/trezor/trezor-firmware/pull/1683/commits/92746053baa1620dd30160814c64f94c213c7413 (I've fixed this up in the wrong commit, I'll fix this during rebasing)